recipe/meta.yaml

```diff
@@ -103,10 +101,8 @@ test:
   source_files:
     - tests
 
-outputs:
-  - type: wheel
-  - name: {{ PKG_NAME }}
```

Comment:
What is this removal?

Reply:
I don't believe this makes any difference.

We don't produce more than one output from this recipe (this counter intuitively suggests we should be?). And the `conda-forge`/`AnacondaRecipes` recipes don't do this.